src/pixelsquiz/buzz.clj

```diff
@@ -17,18 +17,19 @@
   (let [bits (System/getProperty "sun.arch.data.model")]
     (clojure.lang.RT/loadLibrary (str "hidapi-jni-" bits))))
 
-
 (defn open-buzz []
+  (logger/info "Trying to open Buzz controllers")
   (let [_ (load-hid-natives)
-        manager (HIDManager/getInstance)]
```

Comment:
It would be better to try and find the controllers by ID before resorting to finding them by name. The IDs are safer and should cover all controller models whereas names might not.

Also, it's weird that ID matching doesn't work on Linux. Have you looked into why? Does the debug info you added actually print a matching ID?